apps/prairielearn/src/pages/instructorAssessmentQuestions/editQuestionModal.html.ts

```diff
@@ -0,0 +1,184 @@
+import { html } from '@prairielearn/html';
+
+import { Modal } from '../../components/Modal.html.js';
+
+export function EditQuestionModal({
+  newQuestion,
+  question,
+  zoneIndex,
+  questionIndex,
+  alternativeIndex,
+  assessmentType,
+}: {
+  newQuestion: boolean;
+  question?: Record<string, any>;
+  urlPrefix?: string;
```

Comment:
This property appears to be unused.

Reply:
Removed in [3c6791f](https://github.com/PrairieLearn/PrairieLearn/pull/10255/commits/3c6791f5b7ef00282d313997acf9af8418da3660).